.github/workflows/ci.yml

```diff
@@ -27,11 +27,7 @@ jobs:
     - name: Install dependencies
       run: |
         sudo apt-get update --fix-missing
-        sudo apt-get install -y ffmpeg
-    - name: Pip install
-      run: |
-        python -m pip install --upgrade pip
-        pip install -r requirements/prod.txt
+        sudo apt-get install -y ffmpeg python3-scipy
```

Comment:
There is a problem with installing scipy through pip in pypy. For now, just download through apt-get